SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -157,6 +159,24 @@ Each type is a source of requirements that are explicitly identified and formali
 
 #TODO:  Mention traceability & coverage#
 
+==== Requirements Binding Strategies
+Although <<vol1_figure_appendix_a_sdpi_requirements_core_model>> generally indicates bi-directional navigation (arrows on both ends of Requirement-Usage pairs, supporting *_bi-directional bindings_* and navigation is not always helpful.
+This is especially the case when considering potential future updates to the profile specifications.
+In that case, the general rule is:
+
+[none]
+. *Add backward references from _Requirement Usage_ to _Requirement Definition_*.
+
+For example, in TF-2 Transactions, each transaction section is paired with a message transport section in #TF-2 Appendix A MDPWS#; however, future versions of the specification may provide options for alternative transports.
+In this case, the actual transaction definition will remain unchanged, but the bindings to transport messages and services would change.
+Given the rule above, bindings are made in the current TF-2 Appendix A MDPWS profile pointing backweard (or upward!) to the transaction requirements that they satisfy.
+There are no bindings in the opposite direction.
+Taking this approach, a new transport appendix could be added in the future without impacting the core transaction specifications.
+
+Application of this rule would also hold true in other places such as backward references from a profile's Use Case section to the specific #TF-1 Appendix C# use case and scenario requirements that they satisfy.
```

Comment:
Yellow marking? Why not linking TF-1 Appendix C?

Reply:
Updated with the requested link.